tests/integration_tests/test_conjugate_gaussian_models.py

```diff
@@ -73,6 +71,48 @@ def setup_reparam_mask(self, N):
             if torch.sum(mask) < 0.40 * N and torch.sum(mask) > 0.5:
                 return mask
 
+    def model(self, reparameterized, difficulty=0.0):
+        next_mean = self.mu0
+        for k in range(1, self.N + 1):
+            latent_dist = dist.Normal(next_mean, torch.pow(self.lambdas[k - 1], -0.5))
+            mu_latent = pyro.sample("mu_latent_%d" % k, latent_dist)
+            next_mean = mu_latent
+
+        mu_N = next_mean
+        for i, x in enumerate(self.data):
+            pyro.sample("obs_%d" % i, dist.Normal(mu_N, torch.pow(self.lambdas[self.N], -0.5)),
+                        obs=x)
+        return mu_N
+
+    def guide(self, reparameterized, difficulty=0.0):
+        previous_sample = None
+        for k in reversed(range(1, self.N + 1)):
+            mu_q = pyro.param("mu_q_%d" % k, Variable(self.target_mus[k].data +
+                                                      difficulty * (0.1 * torch.randn(1) - 0.53),
+                                                      requires_grad=True))
+            log_sig_q = pyro.param("log_sig_q_%d" % k,
+                                   Variable(-0.5 * torch.log(self.lambda_posts[k]).data +
+                                            difficulty * (0.1 * torch.randn(1) - 0.53),
+                                            requires_grad=True))
+            sig_q = torch.exp(log_sig_q)
+            kappa_q = None if k == self.N \
```

Comment:
Avoid multiline inline `if else` expressions. Just make this an `if else` statement
```py
kappa_q = None
if k == self.N:
    kappa_q = pyro.param(...)
```

Reply:
done